### ISSUE TITLE
fused bn problem when optimize the frozen graph

### ISSUE THREAD

**Issue by ouceduxzk** [ildoonet/tf-pose-estimation#40]
I am trying to optimize my .pb frozen graph and got such warning from tensorflow `WARNING:tensorflow:Incorrect shape for mean, found (0,), expected (24,), for node MobilenetV1/Conv2d_0/BatchNorm/FusedBatchNorm
WARNING:tensorflow:Incorrect shape for mean, found (0,), expected (48,), for node MobilenetV1/Conv2d_1_pointwise/BatchNorm/FusedBatchNorm
WARNING:tensorflow:Incorrect shape for mean, found (0,), expected (96,), for node MobilenetV1/Conv2d_2_pointwise/BatchNorm/FusedBatchNorm
WARNING:tensorflow:Incorrect shape for mean, found (0,), expected (96,), for node MobilenetV1/Conv2d_3_pointwise/BatchNorm/FusedBatchNorm
WARNING:tensorflow:Incorrect shape for mean, found (0,), expected (192,), for node MobilenetV1/Conv2d_4_pointwise/BatchNorm/FusedBatchNorm
WARNING:tensorflow:Incorrect shape for mean, found (0,), expected (192,), for node MobilenetV1/Conv2d_5_pointwise/BatchNorm/FusedBatchNorm
WARNING:tensorflow:Incorrect shape for mean, found (0,), expected (384,), for node MobilenetV1/Conv2d_6_pointwise/BatchNorm/FusedBatchNorm
WARNING:tensorflow:Incorrect shape for mean, found (0,), expected (384,), for node MobilenetV1/Conv2d_7_pointwise/BatchNorm/FusedBatchNorm
WARNING:tensorflow:Incorrect shape for mean, found (0,), expected (384,), for node MobilenetV1/Conv2d_8_pointwise/BatchNorm/FusedBatchNorm
WARNING:tensorflow:Incorrect shape for mean, found (0,), expected (384,), for node MobilenetV1/Conv2d_9_pointwise/BatchNorm/FusedBatchNorm
WARNING:tensorflow:Incorrect shape for mean, found (0,), expected (384,), for node MobilenetV1/Conv2d_10_pointwise/BatchNorm/FusedBatchNorm
`
It seems the mean is not saved properly, any ideas why this happen?  i am using the python scripts tensorflow/python/tools/optimize_for_inference.py to do the transformation

**Comment by ildoonet** [comment]
@ouceduxzk I've just uploaded graph files. I guess they will save your time. 

Closing this issue due to no activity.
